doc/whats_new.rst

```diff
@@ -1580,7 +1581,7 @@ New features
    - The new module :mod:`nilearn.connectome` now has class
      :class:`nilearn.connectome.ConnectivityMeasure` can be useful for
      computing functional connectivity matrices.
-   - The function nilearn.connectome.sym_to_vec in same module
+   - The function :func:`nilearn.connectome.sym_to_vec` in same module
```

Comment:
unrelated change?

Reply:
@jeromedockes This is not the change I made, its happened when I merged my fork with original